setup.cfg

```diff
@@ -9,7 +9,7 @@ show-response = 1
 
 [pytest]
 minversion = 2.3.3
-norecursedirs = ".tox" "build" "docs[\/]_build" "astropy[\/]extern" "astropy[\/]utils[\/]compat"
+norecursedirs = ".tox" "build" "docs[\/]_build" "astropy[\/]extern" "astropy[\/]utils[\/]compat[\/]futures"
```

Comment:
Can you clarify the point of this, @mhvk ? It's not clear to me how (if at all) it relates to the rest of this PR